FIT-27

qrillich opened the pull request:
Добавлены профили Spring.
Добавлено:
application-dev.yml - для свойств профиля "dev"
application-preprod.yml - для свойств профиля "preprod"
Изменено:
application.yml - включен профиль "dev"
Ко всем бинам добавлена аннотация "@profiles"

Commit:
RoleAPI
Добавлено:
RoleController.java - NonNull
RoleDTO.java
IRoleRepository.java
IRoleService.java
RoleService.java
IRoleMapper.java

Необходимо доделать:
-Unit тесты.

Commit:
Merge branch 'devel' into FIT-15

Commit:
Merge branch 'devel' into FIT-15

Commit:
RoleAPI
Изменено:
RoleController.java - mapper убран в сервис
IRoleRepository.java - JavaDOC
IRoleService.java - mapper перенесен из контроллера
RoleService.java - */*

Необходимо доделать:
-Unit тесты.

Commit:
RoleAPI
Изменено:
RoleController.java

Необходимо доделать:
-Unit тесты.

Commit:
Spring Profiles
Добавлено:
application-dev.yml - для свойств профиля "dev"
application-preprod.yml - для свойств профиля "preprod"
Изменено:
application.yml - включен профиль "dev"
Ко всем бинам добавлена аннотация "@Profiles"

Commit:
Spring Profiles
Добавлено:
application-dev.yml - для свойств профиля "dev"
application-preprod.yml - для свойств профиля "preprod"
Изменено:
application.yml - включен профиль "dev"
Ко всем бинам добавлена аннотация "@Profiles"

Commit:
Spring Profiles
Добавлено:
application-dev.yml - для свойств профиля "dev"
application-preprod.yml - для свойств профиля "preprod"
Изменено:
application.yml - включен профиль "dev"
Ко всем бинам добавлена аннотация "@Profiles"

Tema41 committed:
Hot-fix-4 (#24)

* Fit-14
Rest контроллеры для пользователей.
Реализован слой Repository.
Реализован слой service.

* Fit-14
Rest контроллеры для пользователей.
Реализован слой Repository.
Реализован слой service.

* Fit-14
Rest контроллеры для пользователей.
Реализован слой Repository.
Реализован слой service.

* Fit-14
Rest контроллеры для пользователей.
Реализован слой Repository.
Реализован слой service.
Перенес JavaDoc в интерфейс.

* Fit-14

* попробовал удалить файл в гите...

* - Added OpenAPI Swagger v3 dependencies and SpringDoc OpenAPI maven plugin
- Some useful links in FitnessApplication.java and basic @OpenApiDefinition

* Spring Security (#12)

admin/pass

* FIT-10 (#5)

* FIT-10 added users, roles, users_roles tables and entities. New config for Spring in application.yml

* FIT-10 added mappedBy attribute for Role, fixed columns names, remade annotation according to the CodeStyle

* FIT-10 Добавлено: создание последовательности в LB для user id pk; пол в сущности пользователя; аннотации для equals.
Переименованы названия таблиц, полей и файлов в соответсвии с кодстайлом.
Удалён лишний файл changelog, а его содержимое перемещено в оставшийся

* FIT-10 Инициализировал сеты ролей и пользователей хешсетом. Убрал equals и hashcode

Co-authored-by: Роман <Roks420@yandex.ru>

* FIT-16 (#3)

* - Added Recipe Controller, Recipe Service, Recipe Repository, Recipe Entity by Pavel Khramov
- Swagger integration for APIs (some useful annotations and comments)

* - Исправлены методы на ResponseEntity
- Добавлены JavaDoc
- Переименованы классы и интерфейсы
- Убраны ненужные IF

* - Добавлены недостающие Javadoc
- Переведены некоторые комментарии
- @RequestMapping в контроллере (убран повторяющийся код)
- Убран xml для бд

* - Удален Recipe

* - Добавлены комментарии в IRecipeService

* Создание скрипта на таблицу Recipe и сущьности.

* - Починены импорты (#14)

* Добавлено: mapstruct.
Обновлено: org.springdoc 1.6.4 to 1.6.5.
Изменено: вынес версии зависимостей на верх.
Важно: mapstruct плагин не работает с lombok (ошибка инициализации в конструкторе). На колле у кого-то была такая проблема, тут хорошее решение.

* Fit 17 (#4)

* Создание таблицы и сущности Workout. Реализация API и CRUD операций.

* Создание таблицы и сущности Workout. Реализация API и CRUD операций.

* Создание таблицы и сущности Workout. Реализация API и CRUD операций.

* 2nd update: исправил по комментариям ws-eliseev. Пока класс Dto не прописан - использовал Entity

* 2nd update: исправил по комментариям ws-eliseev. Пока класс Dto не прописан - использовал Entity

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил заисимости на mapstruck

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил заисимости на mapstruck

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил зависимости на mapstruck

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил зависимости на mapstruct

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил зависимости на mapstruct

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил зависимости на mapstruct

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил зависимости на mapstruct

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил зависимости на mapstruct

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил зависимости на mapstruct

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил зависимости на mapstruct

* 3rd update: Добавил класс Dto, и директорию util.mapper для Маппера. Добавил зависимости на mapstruct

Co-authored-by: aeliseev <a.s.eliseev@bk.ru>

* Update pom.xml

new line at end

* Добавил .and().csrf().disable(); в SecurityConfig (#22)

* Hot-fix-4

Co-authored-by: Valentin <98877649+ValentPr@users.noreply.github.com>
Co-authored-by: dmitryprikhunov <dp2149@gmail.com>
Co-authored-by: Roman Konchalov <74724989+dxhoan420@users.noreply.github.com>
Co-authored-by: Роман <Roks420@yandex.ru>
Co-authored-by: Artemiy Yahev <90742524+YaSlawaMarlow@users.noreply.github.com>
Co-authored-by: QRillich <kirilliz.nazarov@gmail.com>
Co-authored-by: KensynHimura <90760258+KensynHimura@users.noreply.github.com>
Co-authored-by: aeliseev <a.s.eliseev@bk.ru>
Co-authored-by: QRillich <86483826+QRillich@users.noreply.github.com>

Commit:
HOT-FIX-5 Добавил ко всем необходимым таблицам, аннотациям и последовательностям приставку 'FIT_'
Удалил ненужные файлы: db.changelog-002-create_table_roles.yaml, который забыл удалить, и DEPRECATEDapplication.properties, который уже не нужен, потому что настройки в application.yml, в котором я, кстати, удалил ненужные комментарии.

ws-eliseev commented on src/main/java/ws/eliseev/fitness/controller/RecipeController.java:
```diff
@@ -18,6 +19,7 @@
 @RestController
 @RequiredArgsConstructor
 @RequestMapping("/recipes")
+@Profile("dev")
```
а он тут нужен?

ws-eliseev reviewed (commented):
Вопрос к профилям на контроллерах)

qrillich commented:
> Вопрос к профилям на контроллерах)

Могу предложить использовать профиль по умолчанию для всех бинов, которые будут в любых сборках.

qrillich committed:
Merge pull request #21 from ws-eliseev/FIT-15

FIT-15

Commit:
HOT-FIX-5 Добавил @JsonIgnore в Role.java для предотвращения рекурсии.

dxhoan420 committed:
Merge pull request #26 from ws-eliseev/HOT-FIX-5

HOT-FIX-5 Добавил ко всем необходимым таблицам, аннотациям и последовательностям приставку 'FIT_' и @JsonIgnore для сета пользователей в Role.java добавил

Commit:
Spring Profiles
Изменено:
У контроллеров убраны профили.
Перенес IWorkoutMapper.java в другой пакет.

Commit:
Spring Profiles
Изменено:
Перенес IWorkoutMapper.java в другой пакет.

Commit:
Merge remote-tracking branch 'origin/FIT-27' into FIT-27

# Conflicts:
#	src/main/java/ws/eliseev/fitness/repository/IUserRepository.java

Commit:
Spring Profiles
Изменено:
Удалены профили из контроллеров
Перенес IWorkoutMapper.java в другой пакет.

Commit:
Merge remote-tracking branch 'origin/FIT-27' into FIT-27
